app/views/compute_resources/_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= javascript "compute_resource" %>
 <%= form_for @compute_resource, :html => {:data => {:id => @compute_resource.try(:id)}} do |f| %>
   <%= base_errors_for @compute_resource %>
-
+  <%= alert :class => "alert-success base in fade", :header => _("Success"), :text => _("Test connection was successful")  if defined?(connection_success) && connection_success  %>
```

Comment:
What do you think about moving this to `app/helpers/compute_resources_helper.rb` helper?